src/screens/token-launcher/components/HoldToActivateButton.tsx

```diff
@@ -0,0 +1,166 @@
+import React, { PropsWithChildren, useCallback } from 'react';
+import { StyleProp, TextStyle, ViewStyle } from 'react-native';
+import { triggerHaptics } from 'react-native-turbo-haptics';
+import { Gesture, GestureDetector } from 'react-native-gesture-handler';
+import Animated, { runOnJS, useAnimatedStyle, useSharedValue, withTiming } from 'react-native-reanimated';
+import { ShimmerAnimation } from '@/components/animations';
+import { useBiometryIconString } from '@/components/buttons/BiometricButtonContent';
+import HoldToAuthorizeButtonIcon from '@/components/buttons/hold-to-authorize/HoldToAuthorizeButtonIcon';
+import { useTheme } from '@/theme';
+import { IS_ANDROID } from '@/env';
+import { Box, Text, TextProps } from '@/design-system';
+import { AnimatedSpinner } from '@/components/animations/AnimatedSpinner';
+import { colors } from '@/styles';
+import { useWallets } from '@/hooks';
+import { LedgerIcon } from '@/components/icons/svg/LedgerIcon';
+
+const BUTTON_HEIGHT = 56;
+
+const BUTTON_SCALE_DURATION_IN_MS = 150;
+const LONG_PRESS_DURATION_IN_MS = 500;
+
+type LabelProps = Omit<TextProps, 'children'> & {
+  label: string;
+  showIcon?: boolean;
+  color?: string;
+  testID?: string;
+};
+
+function LabelWithBiometryIcon({ label, showIcon = true, testID, color, ...textProps }: LabelProps) {
+  const { isHardwareWallet } = useWallets();
+  const biometryIcon = useBiometryIconString({ showIcon: !IS_ANDROID && showIcon, isHardwareWallet });
+  const { colors } = useTheme();
+
+  return (
+    <>
+      {/* TODO: */}
```

Comment:
?

Reply:
Sorry forgot to add actual comment I believe this was about the color of the ledger icon being wrong. Shouldn't be blocking though.